site/en/docs/devtools/javascript/source-maps/index.md

```diff
@@ -45,6 +47,10 @@ In {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt
 You might also want to check {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/hmp8j3HiLMCcqPArD9yt.svg", alt="Checkbox.", width="22", height="22" %} **Enable CSS source maps**.
 {% endAside %}
 
+## Check if source maps load sucessfully {: #developer-resources }
+
+See [Developer Resources: View and load source maps manually](/docs/devtools/developer-resources).
+
```

Comment:
Need some changes on the `Debugging with source maps` section. 

In step no 2. enable `group authored/ deployed files` is not a good example here. The screenshot demonstrates a bug. The `index.hash.css` file should not show up in the `Authored` section. 

I check in latest Canary, this issue is gone. CSS show in Deployed. However, the grouping is still not ideal as there are framework generated code show in the Authored section....

Suggest us to remove the step or just update the screenshot

Reply:
@sofiayem step 3 - 7 screenshot s are still incorrect...